src/chef_ci_status.rb

```diff
@@ -6,13 +6,44 @@
 require 'octokit'
 require 'set'
 
-# Get GitHub token
+# Load configuration
+begin
+  require_relative '../config/initializers/config'
```

Comment:
So my instinct here is to say we should use mixlib-config which allows a ruby config file, but ALSO allows yaml/json inherently.

However, after looking, I don't see an easy way to do the `organizations` structure in it. Poke at that and tell me if I'm missing something. If not, we can stick with the custom-built config parsing, but if there's a good way to use mixlib-config I'd prefer that.